cyhy/util/util.py

```diff
@@ -18,20 +18,25 @@
     "report_dates",
     "utcnow",
     "warn_and_confirm",
+    "parse_domains",
 ]
 
-import sys
-import itertools
-import json
-import bson
+# Standard Python Libraries
```

Comment:
Would you please update this list to mirror ow `isort` would output? Also please not that `bson` should go into the third-party section because it is part of the PyMongo library.

```python
from collections import OrderedDict
from datetime import datetime
import itertools
import json
import logging
from logging.handlers import RotatingFileHandler
import sys
import time
```

Reply:
Please see https://github.com/cisagov/cyhy-core/pull/106/commits/fa6b2fa74c9ad5ccc0290c2d54f846beffb569ec.